### DISCUSSION TITLE
Site loading very slow first time

### DISCUSSION THREAD

**Discussion by theooog** [vercel/community#3053] [Bug Report]
### Page to Investigate

tourshark.vercel.app

### Steps to Reproduce

Hello, 

I am in the process of switching my site from netlify to vercel. I got a couple of issues however. The site loads extremely slow, the first time visiting it in a long time. Up to 50 seconds. After this, it loads within 1.2 seconds. 
![Schermafbeelding 2023-06-29 101108](https://github.com/vercel/community/assets/61834257/dc339138-1887-4f3c-9907-fb8ef2b32dee)

I expect the page to load slower because of cold lambda functions, but this is not normal... 

Any ideas what might be going wrong?

**Comment by namernesto** [comment]
I have a similar issue. Not 50 secs long, but def somewhere between 20-30 secs. No idea why
